src/Console/Command/AbstractCommand.php

```diff
@@ -64,6 +65,11 @@
     protected $githubPaginator;
 
     /**
+     * @var \Maknz\Slack\Client
```

Comment:
Fixed, I did an alias to SlackClient to avoid confusion with all other clients on this class. A lot of clients out there